apis/v1alpha1/udproute_types.go

```diff
@@ -52,14 +37,30 @@ type UDPRouteRule struct {
 	// Matches defines which packets match this rule.
 	//
 	// +optional
+	// +kubebuilder:validation:MaxItems=8
 	Matches []UDPRouteMatch `json:"matches"`
```

Comment:
Add `omitempty` tag if the field is optional.

Reply:
Updated, thanks!